sdk/include/reactos/mc/neteventmsg.mc

```diff
@@ -1140,6 +1338,9 @@ Le service %1 ne s'est pas arrêté proprement après avoir reçu un contrôle d
 Language=Polish
 Usługa %1 nie została poprawnie wyłączona po otrzymaniu kodu sterownika przed zamknięciem systemu.
 .
+Language=Romanian
+Serviciul %1 nu a fost oprit corect după primirea controlului de preoprire.
```

Comment:
Another case of reflexive that may fit better than passive, since it's clear that what was expected (but didn't happen) for the given service to gracefully shut itself down.